conda_build/index.py

```diff
@@ -99,16 +99,30 @@ def update_index(dir_path, verbose=False, force=False, check_md5=False):
         json.dump(index, fo, indent=2, sort_keys=True, default=str)
 
     # --- new repodata
+    # Many old packages have outdated index.json file, so we adapt them to fit
+    # the format used in http://repo.continuum.io/pkgs/<LIC>/<PLAT>/repodata.json
     for fn in index:
         info = index[fn]
+
+        # Remove vars that are not used anymore
         for varname in 'arch', 'platform', 'mtime', 'ucs':
             try:
                 del info[varname]
             except KeyError:
                 pass
 
+        # Convert old packages to new format (use 'depends' instead of 'requires'
         if 'requires' in info and 'depends' not in info:
             info['depends'] = info['requires']
 
+        # Remove old format 'requires'. This reproduces index used by
+        info['requires'] = []
+
+        # Fix non-wildcard Python/NumPy dependencies
```

Comment:
Maybe you meant for this whole thing to be indented under the `if 'requires' in info and 'depends' not in info:` line above?